manuscript/frontmatter.html

```diff
@@ -9,7 +9,7 @@
-<p>This work is licensed under a <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/deed.en_US">Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported License</a></p>
+<p>This work is licensed under a <a href="http://creativecommons.org/licenses/by-nc-sa/4.0/">Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License</a></p>
```

Comment:
Other mentions of the license, including the main Readme, referred to the CCA-NC-SA 4.0 license. This updates the front matter to match the other locations; I'm assuming they are the correct version because it's a newer version and there are more occurrences of it.